api/backend.go

```diff
@@ -236,6 +238,16 @@ func (b *StatusBackend) SendTransaction(ctx context.Context, args transactions.S
 	return b.transactor.SendTransaction(ctx, args)
 }
 
+// SignMessage checks the pwd vs the selected account and passes on the metadata
+// to personalAPI for message signature
+func (b *StatusBackend) SignMessage(rpcParams personal.Metadata) sign.Result {
```

Comment:
I would keep `Password` as a separate argument. It's gonna be easier to keep backward compatibility and it will make it clear which part is our extension.

Reply:
Addressed